deps/postject/src/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "postject",
+  "version": "1.0.0-alpha.3",
+  "description": "Easily inject arbitrary read-only resources into executable formats (Mach-O, PE, ELF) and use it at runtime.",
+  "license": "MIT",
+  "engines": {
+    "node": ">=14.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:postmanlabs/postject.git"
+  },
+  "homepage": "https://github.com/postmanlabs/postject#readme",
+  "author": "Postman Labs <help@postman.com> (=)",
+  "bin": {
+    "postject": "./dist/cli.js"
+  },
+  "main": "dist/api.js",
+  "scripts": {
+    "build": "zx ./scripts/build.mjs",
+    "clean": "rimraf ./build",
+    "format": "npm run format:cpp && npm run format:js",
+    "format:cpp": "clang-format -style=chromium -i postject-api.h src/**.cpp test/**.c test/**.cpp",
+    "format:js": "prettier --write src/**.js scripts/**.mjs test/**.mjs",
+    "lint": "npm run lint:cpp && npm run lint:js",
+    "lint:cpp": "clang-format -style=chromium --dry-run --Werror postject-api.h src/**.cpp test/**.c test/**.cpp",
+    "lint:js": "prettier --check src/**.js scripts/**.mjs test/**.mjs",
+    "test": "mocha"
+  },
+  "dependencies": {
+    "commander": "^9.4.0"
```

Comment:
Given that `postject` is going to be vendored in node, it would be a great candidate to use the new `parseArgs` API instead, any thoughts?

Reply:
@ruyadorno Postject currently supports Node.js >= v14 (https://github.com/postmanlabs/postject/blob/f15198b88b7882116c43ff1ea1db969155dcf2d9/package.json#L7) but unfortunately parseArgs has only been backported till v16, so doing this would require dropping support for some versions of Node.js. I'm not particularly opposed to doing that but I don't think such a change would have any impact here because Node.js is probably only going to use the `deps/postject/postject.js` file which is a commonjs module that doesn't have any argument parsing logic. The argument parsing logic stays in `deps/postject/src/dist/cli.js` right now. Anyways, I have moved the Postject addition change to https://github.com/nodejs/node/pull/45066, so feel free to post any concerns regarding this dependency in that PR. Resolving this for now.

Reply:
Node 14 goes EOL in May, so adding major new features for it may not make much sense.

Reply:
I have created an issue for it in postject to track this and I'm personally +1 about this change - https://github.com/postmanlabs/postject/issues/56